### ISSUE TITLE
Have a space for images/gifs

### ISSUE THREAD

**Issue by ianhi** [matplotlib/mpl-third-party#43]
## Problem

I think one of the features of the old third party page is the ability to see images that represent what a library can do. I think this makes scanning much easier if you are just trying to discover what is out there. For instance this:
![image](https://user-images.githubusercontent.com/10111092/115803678-62f8aa00-a3af-11eb-8734-c990697b5899.png)

is pretty tricky for me to get a sense of what each these really do.

## Proposed Solution

I understand that there shouldn't be images on the main third party page, but what if when you clicked on the headings it took you to a subpage that included images/gifs? 

Or perhaps a button that could be clicked to expand and show an image/gif

**Comment by story645** [comment]
I'm all for an alternate page that's a gallery view - like toggling between the two - can probably build it off the same yaml file...

**Comment by jklymak** [comment]
Except you are making submission as heavy a burden as it is currently.  We have a link to their website, which presumably has an image...

**Comment by ianhi** [comment]
I'm not sure I fully understand the argument re burden. If they already have an image then shouldn't be so hard to add one?

But more generally: Was the purpose of this repo to make it "less of a burden" to submit a third party package? I would have guessed that it was to reduce the noise and repo size for the main repo. As it stands I don't see how this is significantly easier to submit to than the previous method, although I'll admit that a yml file is easier to edit than a rst file and theres no mucking around with styling. 

Also I suppose I'm proposing the image as an optional addition

**Comment by jklymak** [comment]
1. Easy to edit yml
2. No rebases because they are just adding an atomized file. 
3. Quick turn around.  No need to build all the docs for this page to be updated. 
4. No back and forth about image size (see every recent PR that added a new third party). 

We could have a drill down page if someone wants to build it.  But I'd argue we should get the basics down first.

**Comment by story645** [comment]
I think adding an image becomes way less of a back and forth if we can clearly put the size restrictions + link to optimization tool as part of the add library instructions. Currently it's part of the back and forth b/c partially because it's not even stated in the first place. 
Also, there are a bunch of github actions that automatically optimize/compress images, and this being a stand along repo makes it easier to add actions specific to the repo:
* https://github.com/marketplace/actions/image-optimizer
* https://calibreapp.com/blog/compress-images-in-prs
****

**Comment by QuLogic** [comment]
With the new style, we have all the power of Bootstrap now (e.g., with placeholders):
![image](https://user-images.githubusercontent.com/302469/115821016-f80b9b00-a3cf-11eb-9ebc-300e5e54df6e.png)

**Comment by jklymak** [comment]
Yes, but just to remind folks of the motivation, its to have hundreds of these, not the 20 or so we have now.  

The point of having this in a yml database is that we can organize and format at will, without involving the original authors.  

Perhaps that will be compatible with storing, serving and displaying images, but thats more complication.
